release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -69,6 +69,11 @@ You can also specify multiple regions and zones on a cluster after installation.
 ==== Changes to the default `install-config.yaml` file
 After you run the installation program for {product-title} on vSphere, the default `install-config.yaml` file now includes `vcenters` and `failureDomains` fields, so that you can choose to specify multiple datacenters, region, and zone information for your cluster. You can leave these fields blank if you want to install an {product-title} cluster in a vSphere environment that consists of single datacenter running in a VMware vCenter.
 
+[id="ocp-4-13-installation-and-upgrade-three-node"]
+==== Three-node cluster support
+
+Beginning with {product-title} {product-version}, deploying a three-node cluster is supported on Amazon Web Services(AWs),  Microsoft Azure, Google Cloud Platform (GCP), and VMware vSphere. This type of {product-title} cluster is a smaller, more resource efficient cluster, as it only consists of three control plane machines, which also act as compute machines.
```

Comment:
```suggestion
Beginning with {product-title} {product-version}, deploying a three-node cluster is supported on Amazon Web Services (AWS), Microsoft Azure, Google Cloud Platform (GCP), and VMware vSphere. This type of {product-title} cluster is a smaller, more resource efficient cluster, as it consists of only three control plane machines, which also act as compute machines.
```
- In the preview, {product-version} renders as "Branch Build". I assume that variable will be updated by the time this goes live?
- Per IBM Style, place "only" as close to the word it modifies as possible. (https://www.ibm.com/docs/en/ibm-style?topic=grammar-adverbs-only)

Reply:
That's right @lahinson. The variable is replaced with `4.13` when it goes live. Thanks for the review.